aireview/src/aireview/domain/services/code_analyzer.py

```diff
@@ -16,10 +17,9 @@ def analyze_code(self, pull_request: PullRequest) -> Review:
         # self._ai_agent.initialize(f"Creating context for review of pull request {pull_request.number}")
         for index, file in enumerate(pull_request.files):
             prompt = self._build_analysis_prompt(file)
-            print(f"prompt: {prompt}")
             analysis = self._ai_agent.analyse(prompt)
-            print(f"analysis: {analysis}")
-            review_comments.append(self._parse_analysis(analysis))
+            comments = self._parse_analysis(analysis)
+            review_comments.extend(comments)
```

Comment:
**style**: Hard-coded values in Review constructor (1, 'summary of review') should be derived from analysis results or made configurable